src/types/madie-madie-util.d.ts

```diff
@@ -11,12 +11,10 @@ declare module "@madie/madie-util" {
   }
 
   interface FeatureFlags {
-    includeSDEValues: boolean;
     manifestExpansion: boolean;
```

Comment:
I think this needs to be removed

Reply:
and any other place that might be using it. have seen something in `useTerminologyServiceApi`